content/_data/changelogs/lts.yml

```
@@ -9733,6 +9733,506 @@
     message: |-
       Restore context menus of model links in build history views and administrative monitors (regression in 2.402).
 
+- version: "2.426.1"
+  date: 2023-11-15
+  banner: >
+      Short tags (without "jdk" in them) such as <code>jenkins/jenkins:2.414.3-alpine</code> are using Java 17 and not Java 11 like previously.
+      If you need to keep using Java 11, use tags like <code>jenkins/jenkins:2.414.3-jdk11</code>.
+      Also note that two new tags (2.414.3-alpine-jdk17 & 2.414.3-slim-jdk17) have been published without any content change a week later than the original ones.
```

The last sentence applies to 2.414.3 but does not apply to 2.426.1, so I removed it.  I've done some optional rephrasing of the first two sentences, mostly based on personal preferences.  You're welcome to reject changes in the first two sentences.  I'd really like the last sentence removed.
```suggestion
      Short container image tags (without "jdk" in them) such as <code>jenkins/jenkins:2.426.1</code> are now using Java 17.
      If you need to continue using Java 11, use tags like <code>jenkins/jenkins:2.426.1-jdk11</code>.
```

---

That makes sense, and I didn't catch it, so thanks very much @MarkEWaite!